jaxrs/pom.xml

```diff
@@ -107,29 +107,6 @@
             <artifactId>kafka-cdi-extension</artifactId>
```

Comment:
Maybe not part of this task but it might be good to remove kafka dependencies when we have cdi-extension dependency. @dimitraz what do u think?